src/martialarts.cpp

```diff
@@ -1711,15 +1745,22 @@ bool ma_style_callback::key( const input_context &ctxt, const input_event &event
                     continue;
                 }
                 weaps.second.erase( std::unique( weaps.second.begin(), weaps.second.end() ), weaps.second.end() );
-                std::string w_cat( weaps.first );
+                std::string w_cat;
+                if( weaps.first.is_valid() ) {
+                    w_cat = weaps.first->name().translated();
+                } else {
+                    // MISSING JSON DEFINITION intentionally not translated
+                    w_cat = weaps.first.str() + " - MISSING JSON DEFINITION";
+                }
+
                 std::replace( w_cat.begin(), w_cat.end(), '_', ' ' );
```

Comment:
I think this is no longer necessary.

Reply:
I only put in there because the ID is very similar or equal to the name. Having the `MISSING JSON DEFINITION` there warns that it is not translated ( as I'm not expecting translation bug reports about it ).

Reply:
Actually I mean line 1756
```cpp
std::replace( w_cat.begin(), w_cat.end(), '_', ' ' );
```

I think we don't need to substitute anything in translated category names.